.github/workflows/cd.yml

```diff
@@ -34,13 +36,14 @@ jobs:
         uses: 'phips28/gh-action-bump-version@master'
         with:
           tag-prefix: 'v'
+          commit-message: 'CI: bumps version to {{version}} [skip ci]'
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.ACTION_AUTH_TOKEN }}
```

Comment:
why is the necssary? can we just use this? 

https://github.com/decentralized-identity/JWS-Test-Suite/blob/main/.github/workflows/ci.yml#L32

We should not need secrets for this IMO, the default github tokens should be enough.

Reply:
The default github tokens cannot push to protected branches